middleware/metadata/metadata_test.go

```diff
@@ -137,3 +137,24 @@ func Test_WithPropagatedPrefix(t *testing.T) {
 		t.Error("The prefix must be overridden.")
 	}
 }
+
+func Test_hasPrefix(t *testing.T) {
```

Comment:
TestHasPrefix

Reply:
Golang does not recommend this naming method, and the standard library does not have this naming method